libc/include/llvm-libc-macros/EFIAPI-macros.h

```diff
@@ -0,0 +1,10 @@
+#ifndef LLVM_LIBC_MACROS_EFIAPI_MACROS_H
```

Comment:
This header is missing the copyright header.

Reply:
Added